engine/src/main/java/com/datatorrent/stram/StreamingAppMasterService.java

```diff
@@ -138,6 +139,7 @@
    * This should be replaced when a constant is defined there
    */
   private static final String SSL_SERVER_KEYSTORE_LOCATION = "ssl.server.keystore.location";
+  private static final int NODE_MANAGER_KILL_CONTAINER_TIMEOUT = 30 * 1000;
```

Comment:
Can you make it configurable by a system property. See bufferserver.server.Server.BACK_PRESSURE_ENABLED